### PR TITLE
[time.format] Add/fix region-index entry for "format"

### PR DESCRIPTION
**Pull request by tkoeppe**
I can't tell what the perceived problem behind the `FIXME` was; I'd welcome feedback; @AlisdairM?

### CODE REVIEW

File: source/time.tex [hunk 1]
```diff
@@ -10417,6 +10417,8 @@
 \end{itemdescr}
 
 \rSec1[time.format]{Formatting}
+\indexlibrary{\idxcode{format}|)}%
+%% See also FIXME below
```
Comment (jensmaurer) [inline source/time.tex]: Please drop the FIXME; if there is an issue, a github issue is the right procedure to track it.

Comment (tkoeppe) [reply source/time.tex]: Done. It was partially to serve as a review hint to solicit suggestions for a better indexing range.

File: source/time.tex [hunk 2]
```diff
@@ -10938,7 +10939,6 @@
 };
 \end{codeblock}
 
-% FIXME: Decide how to index this. This template-id is a lie.
```
Comment (jensmaurer) [inline source/time.tex]: Now you've removed this totally unrelated FIXME.

Could I ask you limit this change to just the indexing snafu? Thanks.

**Comment by jensmaurer** [comment]
Ah, that was where the "see below" pointed to. Anyway, please create separate github issues for any follow-up tasks.

File: source/time.tex (hunk 2 above)
Comment (tkoeppe) [reply source/time.tex]: This _is_ the snafu. It's a previously known one, just not one that was previously resolved...

**Pull request merged**